code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -435,5 +435,6 @@ var/global/list/lawlorify = list (
 	soulCounter = SOULVALUE
 	var/obj/screen/devil/soul_counter/K = new()
 	K.update()
+	var/mob/living/carbon/D = new()
```

Comment:
You're a freaking mvp. Only problem is that says owner.current.handle_devil is an undefined proc, but I think I can figure it out myself